solidity/test/bridge/Bridge.test.ts

```diff
@@ -4849,6 +4849,833 @@ describe("Bridge", () => {
+                  )
+
+                await increaseTime(await bridge.redemptionTimeout())
+
+                initialPendingRedemptionsValue = (
+                  await bridge.getWallet(data.wallet.pubKeyHash)
+                ).pendingRedemptionsValue
+
+                initialRedeemerBalance = await bank.balanceOf(
+                  data.redemptionRequests[0].redeemer
+                )
+
+                const redemptionKey = buildRedemptionKey(
+                  data.wallet.pubKeyHash,
+                  data.redemptionRequests[0].redeemerOutputScript
+                )
+
+                redemptionRequest = await bridge.pendingRedemptions(
+                  redemptionKey
+                )
+
+                tx = await bridge
+                  .connect(thirdParty)
+                  .notifyRedemptionTimeout(
+                    data.wallet.pubKeyHash,
+                    data.redemptionRequests[0].redeemerOutputScript
+                  )
+              })
+
+              after(async () => {
+                await restoreSnapshot()
+              })
+
+              it("should update the wallet's pending redemptions value", async () => {
+                const expectedPendingRedemptionsValue =
+                  initialPendingRedemptionsValue
+                    .sub(data.redemptionRequests[0].amount)
+                    .add(redemptionRequest.treasuryFee)
+
+                const currentPendingRedemptionsValue = (
+                  await bridge.getWallet(data.wallet.pubKeyHash)
+                ).pendingRedemptionsValue
+
+                expect(currentPendingRedemptionsValue).to.be.equal(
+                  expectedPendingRedemptionsValue
+                )
+              })
+
+              it("should return the requested amount of tokens to the redeemer", async () => {
+                const expectedRedeemerBalance = initialRedeemerBalance.add(
+                  data.redemptionRequests[0].amount
+                )
+                const currentRedeemerBalance = await bank.balanceOf(
+                  data.redemptionRequests[0].redeemer
+                )
+                expect(currentRedeemerBalance).to.be.equal(
+                  expectedRedeemerBalance
+                )
+              })
+
+              it("should remove the request from the pending redemptions", async () => {
+                const redemptionKey = buildRedemptionKey(
+                  data.wallet.pubKeyHash,
+                  data.redemptionRequests[0].redeemerOutputScript
+                )
+                const request = await bridge.pendingRedemptions(redemptionKey)
+
+                expect(request.redeemer).to.be.equal(
+                  "0x0000000000000000000000000000000000000000"
+                )
+                expect(request.requestedAmount).to.be.equal(0)
+                expect(request.treasuryFee).to.be.equal(0)
+                expect(request.txMaxFee).to.be.equal(0)
+                expect(request.requestedAt).to.be.equal(0)
```

Comment:
It's enough to check just if `requestedAt` is zero.

Reply:
Done in https://github.com/keep-network/tbtc-v2/commit/a769909bce54058695ce018258de2ada00bb6175.